lib/matplotlib/backends/__init__.py

```diff
@@ -14,22 +14,21 @@
            'new_figure_manager', 'backend_version']
 
 backend = matplotlib.get_backend() # validates, to match all_backends
+if backend.startswith('module://'):
+    backend_name = backend[9:]
+else:
+    backend_name = 'matplotlib.backends.backend_' + backend.lower()
 
-def pylab_setup():
-    'return new_figure_manager, draw_if_interactive and show for pylab'
+def get_backend():
```

Comment:
@fariza yes, I put the setter as the basic idea... At the moment we have ``matplotlib.pyplot.switch_backend()``

I have written the code with the possibility to:
1. Add a setter method for ``FigureManager.backend`` so we can apply the ``switch_backend()`` code to a specific figure.
2. use the pyplot method to switch the backend only for future backends.

Not too convinced about the need to move the module method ``get_backend`` up further up, as we already have ``switch_backend``